Dockerfile.mockccc

```diff
@@ -7,13 +7,32 @@ ARG BUILDNUM=""
 FROM scrolltech/go-rust-builder:go-1.20-rust-nightly-2022-12-10 as builder
 
 ADD . /go-ethereum
-RUN cd /go-ethereum && env GO111MODULE=on go run build/ci.go install ./cmd/geth
+
+RUN mkdir -p /scroll/lib/
+
+COPY --from=zkp-builder /app/target/release/libzkp.so /scroll/lib/
```

Comment:
What's `zkp-builder`? I'm getting this error:

```
ERROR: failed to solve: zkp-builder: failed to resolve source metadata for docker.io/library/zkp-builder:latest: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```

Reply:
I think @colinlyguo copies by mistake from `Dockerfile` to `Dockerfile.mockccc`

BTW we don;t need `libzkp` in  `Dockerfile.mockccc`.   ccc is expected disabled in mockccc

Reply:
oh sorry, have not tested after last change. fixed in [abf2b89](https://github.com/scroll-tech/go-ethereum/pull/726/commits/abf2b89f6e407e938495da5785e3138896abd758).

try building now.

Reply:
it works after adding this commit: [d71701c](https://github.com/scroll-tech/go-ethereum/pull/726/commits/d71701cc364e1eaa1a9aaf3cb0f8ff1cde4aa579)